src/api.js

```diff
@@ -35,3 +35,19 @@ export const unlikeReview = (review_id) => {
     return res.data.review;
   });
 };
+
+export const postComment = (comment, review_id) => {
+  const inputComment = { username: "tickle122", body: comment };
```

Comment:
The username should be pulled from the currently logged in user. I know Cnmoosavinia doesn't exist in the database at the moment, but you may want to add him in the backend to allow this functionality to be demonstrated on the front-end.

Reply:
accessed api to get list of users then set current user to position [1] in the user array